hw07_file_copying/go.mod

```
@@ -1,3 +1,11 @@
-module github.com/fixme_my_friend/hw07_file_copying
+module github.com/DimVlas/hw07_file_copying
```

```suggestion
module github.com/DimVlas/otus_hw/hw07_file_copying
```
Следует указать полный путь до репозитория